.gitignore

```diff
@@ -170,3 +170,6 @@ knip.json
 # Schemathesis temporary files
 .hypothesis/
 .cassettes/
+
+# Locally generated keys
+*.key
```

Comment:
Hmm, wonder if this is actually helpful or if it's gonna mess with our ability to commit test keys 😅

Reply:
removed